1136 - Cleanup pages (Round Two) (#1498)
infor-design/enterprise: src/components/column/readme.md

## Code Example

> This example shows how to invoke a simple column chart with a dataset switch a value for each column. You can pass in a tooltip with a custom value, and also pass in 3 labels that are used at the various break points as will fit (name, showName, abbrName).

Reviewer: ```suggestion
This example shows how to invoke a simple column chart with a dataset with a value for each column. You can pass in a tooltip with a custom value, and also pass in three labels that are used at the various break points as will fit (name, showName, abbrName).
```

Reviewer: I assume this is "with" and not :"switch"?

$('#column-bar-example').chart({type: 'column', dataset: dataset});
```

## Selection Model

> - You can initially mark a bar on the chart as selected by passing selected: true in the dataset element.

Reviewer: ```suggestion
- You can initially mark a bar on the chart as selected by passing `selected: true` in the dataset element.
```

```javascript
{
  name: 'Automotive',
  shortName: 'Auto',
  abbrName: 'A',
  value: 7,
  selected: true
}
```

- You can use the `getSelected` method to get the currently selected line.

> - You can use the `setSelected` method to set the current selected line. For this function you can pass in options such as groupIndex, fieldName and fieldValue to find the associated values.

Reviewer: ```suggestion
- You can use the `setSelected` method to set the current selected line. For this function you can pass in options such as groupIndex, fieldName and fieldValue to find the associated values.
```
